tests/data/config/n.py

```diff
@@ -21,3 +21,5 @@ def func(x):
     f=dict(a='69'))
 dict_item5 = {'x/x': {'a.0': 233}}
 dict_list_item6 = {'x/x': [{'a.0': 1., 'b.0': 2.}, {'c/3': 3.}]}
+dict_item7 = osp.expanduser('~')
+dict_item8 = func('a')
```

Comment:
It could be better to add some unit tests to test could `Config` dumps the windows path successfully.